easy_phi/app.py

```diff
@@ -200,8 +213,9 @@ def post(self):
         """Transfer SCPI command to a module and return the response"""
         # Check user lock status
         used_by = getattr(self.module, 'used_by', None)
+        print used_by
         # auth.user_by_token(None) returns None, in case selection isn't used
-        if used_by != auth.user_by_token(self.api_token):
+        if used_by and used_by != auth.user_by_token(self.api_token):
```

Comment:
Without this SCPI commands handler was failing with 409 every time. Did not go into too much details why this is happening (99% sure it's related to my temporary assignment of 'temporary_token' to each user as API token). You are working on authentication so I assume this will be removed by your work